libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -88,6 +89,10 @@ export abstract class BaseInstallAction extends BaseRushAction {
       parameterLongName: '--ignore-hooks',
       description: `Skips execution of the "eventHooks" scripts defined in rush.json. Make sure you know what you are skipping.`
     });
+    this._offlineParameter = this.defineFlagParameter({
+      parameterLongName: '--offline',
+      description: `Do not attempt to access the network. Report an error if the required dependencies cannot be obtained from the local cache.`
```

Comment:
This implies that Rush will definitely never try to make a network request, which isn't strictly true.

Reply:
Good feedback. I have improved the description.